sim/CMakeLists.txt

```diff
@@ -1,37 +1,44 @@
 cmake_minimum_required(VERSION 3.16)
+include(FetchContent)
+# show progress for when fetching FetchContent dependencies
+set(FETCHCONTENT_QUIET OFF)
+# use new policy for options
+set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
```

Comment:
What kind of options?
Would be good to add in comments too

Reply:
Added